### ISSUE TITLE
Collector crystals do not work Astral Sorcery

### ISSUE THREAD

**Issue by RayCore** [Darkosto/SevTech-Ages#1954]
I have added only aroma backup, which should not affect this, and i didn't find(probably blind) an issue about this nor here nor on the astral sorcery repo

Basically no collector crystal works, both normal and celestial, i first used a 900/100/100 armara attuned collector crystal, it didn't work at all, not even a little, so i cheated other attuned collectors to see if the problem was the constellation, it wasn't. Even if i correctly build a enhancing structure it won't work
But ONE DAY. only one, it seemd to work, i don't know why...

## Steps to Reproduce (for bugs)
1. Create a celestial/iridiscent altar with the proper multiblock
2.Link a collector crystal to it

## Logs
All the logs(4 links cause, muh, 512KB limit):
https://pastebin.com/fUweQ5Ma
https://pastebin.com/z7w588xZ
https://pastebin.com/XPNYsPg0
https://pastebin.com/vhnRijZk

the part when this bug is in the last pastebin right a the end before i close the world

## Client Information
* Modpack Version: 3.0.6
* Java Version: 1.8.0.171
* Launcher Used: Twitch app
* Memory Allocated: 8192 MB
* Resourcepack Enabled?: No
* Optifine Installed?: No

Hope i didn't forget anything.

**Comment by maverickx29** [comment]
do you have a picture of you setup? they need line of sight to their destination. additionally if the constellation is not in the sky you get less out of the collector, but it should still work and link. mine works fine, so maybe double check it.

**Comment by RayCore** [comment]
Here it is: https://imgur.com/a/3PkGc8k
the amount of starlight does not change wheter the collector is linked or not

**Comment by maverickx29** [comment]
they don't show up for me either, its a bit frustrating and likely a bug, but you can check the telescope to see if the constellation is in the sky. if it is and it still isn't doing anything my only other thought is your in a dead zone where not much starlight is hitting. in which case when armara is in the sky and the one its conflicting with is not was the one day it worked, that is more of a guess though. i'm not really sure beyond that.

**Comment by Rixxaw** [comment]
I had a similar issue.  I even tried 2 crystals thinking I needed more to boost it.  My table was in a Starfield exactly centered on the blue cloud.  I was very frustrated and tried everything to fix it.  Finally as a last ditch before tearing it all down and moving it up to Y 121 I decided to put the crystal on the other side of the altar just to to see if that made a difference, and boom Full Table 24x7x365.  I checked the Chunk boundaries and apparently the two previous crystals were not in the same chunk as the table.  When I moved it, I moved into the same chunk as the table.  I reported it as a bug to Astral and they said it wasn't the chunk, but I'm still convinced it was the chunk.  The Dev said that there is a really old bug with Forge where some Chunks aren't working properly and maybe I was hitting that bug.  I think with Sevtech being such a heavy modpack.  Its likely we are hitting a lot of bugs that haven't been seen in a long time.  Check your crystal is in the same chunk as your table, if it is just move it to another spot and see if that fixes it for you.
